specs/proposals.md

```diff
@@ -24,13 +27,23 @@ are part of later specification milestones.
 
```

Comment:
Reminds me I have a local branch where I attempted to reword this document!
Let's get this merged, and I'll rebase on top of that and make another PR.